src/ast/ast.h

```diff
@@ -364,6 +364,7 @@ class Builtin : public Node {
   std::string ident;
   int probe_id;
   SizedType builtin_type;
+  bool available = true;
```

Comment:
If we do the check in an earlier pass and replace the node, we don't need this added variable.